app/src/main/kotlin/com/simplemobiletools/clock/extensions/Context.kt

```diff
@@ -168,7 +168,16 @@ fun Context.scheduleNextWidgetUpdate() {
 
     val alarmManager = getSystemService(Context.ALARM_SERVICE) as AlarmManager
     val triggerAtMillis = System.currentTimeMillis() + getMSTillNextMinute()
-    alarmManager.setExact(AlarmManager.RTC, triggerAtMillis, pendingIntent)
+
+    when {
+        Build.VERSION.SDK_INT >= Build.VERSION_CODES.M -> {
```

Comment:
use isMarshmallowPlus() please. Also use such splitting into lines with brackets only in case there are multiple things happening on the same line, like checking some more complicated condition. If there will be a crash at such lines, it is important to know at which code did th crash happen. Such easy cases can be simplified to a single line without brackets.